deployment/proxy/docker-compose-proxy.yml

```diff
@@ -0,0 +1,44 @@
+services:
+  aerie_proxy:
+    container_name: aerie_proxy
+    depends_on:
+     - "aerie_ui"
+     - "hasura"
+     - "aerie_gateway"
+    build:
+      context: proxy
+    ports:
+     - "80:80"
+     - "443:443"
+     - "8080:8080"
+     - "9000:9000"
+    environment:
+        NGINX_ENVSUBST_TEMPLATE_SUFFIX: ".template"
+        AERIE_HOST: ${AERIE_HOST}
+    networks:
+      - aerie_net
+  aerie_gateway:
+    ports: !reset []
```

Comment:
What does `!reset []` do?

Reply:
This is a newer feature of docker compose, that let's you merge together multiple `docker-compose.yml` files. The `!reset[]` directive tell the merge algorithm to completely override the ports section of our regular `deployment/docker-compose.yml`, i.e. it closes the previously exposed ports, since `nginx` handles them now.

https://docs.docker.com/compose/multiple-compose-files/merge/